Dockerfile

```diff
@@ -1,24 +1,30 @@
 FROM fedora:rawhide
-MAINTAINER Pavel Odvody <podvody@redhat.com>
+MAINTAINER Jaroslav Mracek <jmracek@redhat.com>
 
-RUN dnf -y install automake cmake dnf-plugins-core expat-devel \
-                   gcc-c++ git make python-behave rpm-build wget which\
- && dnf -y builddep rpm libsolv\
- && rm -rf /var/cache/dnf/*
+RUN dnf -y install dnf-plugins-core
 
-RUN git clone https://github.com/rpm-software-management/rpm.git\
- && (cd rpm/\
-  && ./autogen.sh --prefix=/usr --with-external-db --enable-python CPPFLAGS="-I/usr/include/nspr4 -I/usr/include/nss3"\
-  && make && make install && rm -rf /rpm/)
+#RUN dnf -y upgrade dnf
 
-RUN git clone https://github.com/openSUSE/libsolv.git\
- && (cd libsolv/ && mkdir build && cd build/\ 
-  && cmake -DENABLE_LZMA_COMPRESSION=1 -DENABLE_COMPLEX_DEPS=1 -DUSE_VENDORDIRS=1 -DFEDORA=1 -DCMAKE_INSTALL_PREFIX=/usr ../\
-  && make && make install && rm -rf /libsolv/)
+RUN dnf -y upgrade rpm
+
+RUN dnf -y upgrade libsolv
+
+RUN dnf -y install httpd python-behave
+
+COPY repo /var/www/html/repo/
+
+ADD httpd.conf /etc/httpd/conf/
 
 ADD launch-test /usr/bin/
-VOLUME /repo 
 
-RUN echo -ne "[test]\nname=test\nbaseurl=file:///repo\nenabled=1\ngpgcheck=0" > /etc/yum.repos.d/test.repo
+RUN mkdir -p /temp/dnf.repo
```

Comment:
This dir will be used by behave script to store repository from yum.repo.d. All of repository will be moved to this directory.